exporter/awsemfexporter/datapoint.go

```diff
@@ -224,6 +224,8 @@ func getDataPoints(pmd *pdata.Metric, metadata CWMetricMetadata, logger *zap.Log
 		return
 	}
 
+	// We only calculate delta when metrics are generated for container insights prometheus
+	isPrometheusMetric := metadata.receiver == prometheusReceiver
```

Comment:
Would this be a good idea to make it configurable rather than assuming it as default behavior? There may have customer who uses prometheus receiver and doesn't want this adjustment.

Reply:
`prometheus` is an attribute injected by the otel config used by container insights. By default, there's no such attribute. The adjustment will not be triggered for other users who use prometheus receiver.

Reply:
Then I feel like we can change the name to be more meaningful. The attribute `prometheus` as well as this variable name. Something like `prometheusForContainerInsights`.

Reply:
I'm afraid I can't change the attribute value cuz it's already used by our existing cloud watch agent customers and we need to take care of the compatibility during the migration.

Reply:
Done

Reply:
Thanks.